src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/Microsoft.Azure.Management.MachineLearningCompute.csproj

```diff
@@ -4,9 +4,9 @@
         <PackageId>Microsoft.Azure.Management.MachineLearningCompute</PackageId>
         <Description>Provides developers with a library to create and manage machine learning compute resources.</Description>
         <AssemblyName>Microsoft.Azure.Management.MachineLearningCompute</AssemblyName>
-        <Version>0.1.0</Version>
+        <Version>1.0.0</Version>
```

Comment:
Major version bump.

Reply:
Question about version bumps in general... We're in public preview so we're still changing quite a few things and might have quite a few breaking changes. Is there some way around this so that once we're in GA this library isn't already a high version?

Reply:
if you are in preview, then ignore what I said. You don't have to update your major version for previews, you can simply bump up minor version.